Swift/ReplicatorConfiguration.swift

```diff
@@ -302,6 +393,26 @@ public struct ReplicatorConfiguration {
     
     // MARK: Internal
     
+    var defaultCollectionConfig: CollectionConfiguration {
+        guard let col = try? self.database.defaultCollection() else {
+            Database.throwNotOpenError()
```

Comment:
No need to check here as it has already been checked in initCollectionConfigs.

Reply:
Even if the default collection is present when we created the replicator config, but it can be invalid when it set properties(channels, documentIDs, filters etc..) later. Also swift optionals will make us check for empty when default collection is fetched, as well as collection config.